src/components/iconbutton.js

```diff
@@ -25,21 +26,35 @@ export class IconLinkButton extends LitElement {
     href: { attribute: true },
     alt: { attribute: true },
     icon: { attribute: true },
+    onClicked: { attribute: true },
   };
   constructor() {
     super();
     this.href = "";
     this.alt = "";
     this.icon = "";
+    this.onClicked = "";
   }
   onClick() {
-    browser.tabs.create({
-      url: this.href,
-    });
+    switch (this.onClicked) {
+      case "open-link":
+        browser.tabs.create({
+          url: this.href,
+        });
+        break;
+      case "go-back":
```

Comment:
Wondering if we need a custom event vs just setting href to null, (checking that in onclick) and just have the parent component have an  `@click=${goBack()}` listener.